libraries/ui/src/ui/types/ContextAwareProfile.cpp

```diff
@@ -47,11 +95,10 @@ bool ContextAwareProfile::isRestrictedInternal() {
     return true;
 }
 
+void ContextAwareProfile::onIsRestrictedChanged(bool newValue) {
```

Comment:
maybe `setRestricted` instead?

Reply:
This is called by the notification broadcaster and strictly from within the object, so setRestricted feels less appropriate.

Reply:
it’s called from the static method (otherwise, since it’s just called in one place, I’d recommend getting rid of the method entirely).  but per https://github.com/kasenvr/project-athena/blob/kasen/core/CODING_STANDARD.md#121-the-terms-getset-must-be-used-where-an-attribute-is-accessed-directly it should still be setRestricted

Reply:
I can see that name being confused by anyone looking at it as an API.  Since it's simplified down to a simple variable assignment at this point I'm tempted to just change it to a direct variable assignment and get rid of the function then.